src/components/sky-nav/sky-nav.twig

```diff
@@ -0,0 +1,40 @@
+{# https://webaim.org/techniques/skipnav/ #}
+<a href="#main">
+  Skip to main content
+</a>
+{#
+  This may require `role="banner"` if nested in something other than `body`.
+  https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Banner_Role
+#}
+<header>
+  <a href="/"
+    {% if is_home %}
+      aria-current="page"
+    {% else %}
+      title="Return to home"
+    {% endif %}>
+    {# https://heydonworks.com/article/aria-label-is-a-xenophobe/ #}
+    <span class="u-hidden-visually">Home: Cloud Four</span>
+    {# http://simplyaccessible.com/article/7-solutions-svgs/#acc-heading-4 #}
+    {% include '../../assets/icons/cloud-four.svg.twig' with {
+      width: '80',
+      height: '80',
+      focusable: 'false'
+    } only %}
+  </a>
+  <button type="button">
+    <span class="u-hidden-visually">Toggle Main</span>
+    Menu
+  </button>
+  <nav aria-label="Main Menu">
+    <ul>
```

Comment:
I've been curious lately about skipping the unordered list and directly inserting links into a nav, but [this comment by expert Léonie Watson](https://css-tricks.com/navigation-in-lists-to-be-or-not-to-be/#comment-273913) convinced me we should keep the unordered list.

> A quick straw poll of around 30 screen reader users this afternoon suggests that many find lists useful in this context. Some said a list may not be nescessary if a heading were present before the navigation block, and some said they preferred to use the features of their screen reader to access links but acknowledged that if they were looking for a specific group of links that wouldn’t help.

I'm assuming that when we get to CSS we'll be adding `list-style: none;` to this. With that in mind, we should maybe add `role="list"` to [maintain list semantics in Safari](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html).